src/main/java/org/opentripplanner/graph_builder/module/AddTransitModelEntitiesToGraph.java

```diff
@@ -33,8 +48,8 @@
 
     private final OtpTransitService transitService;
 
-    // Map of stops and their vertices in the graph
-    private Map<Stop, TransitStopVertex> stopNodes = new HashMap<>();
+    // Map of all gtfs stops and their vertices in the graph
+    private Map<TransitEntity<FeedScopedId>, Vertex> stopNodes = new HashMap<>();
```

Comment:
This is not just for GTFS, it also apply to the Netex import, so the original comment is probably better.